models/index.js

```diff
@@ -39,3 +40,43 @@ exports.fetchCommentsByReviewId = (id) => {
     return comments;
   });
 };
+
+exports.addCommentByReviewId = (id, commentObj) => {
+  const { username, body } = commentObj;
+  if (!username) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request: Missing username property",
+    });
+  }
+  if (!body) {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request: Missing body property",
+    });
+  }
+  if (typeof username !== "string") {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request: Incorrect data type on username",
+    });
+  }
+  if (typeof body !== "string") {
+    return Promise.reject({
+      status: 400,
+      msg: "Bad Request: Incorrect data type on body",
+    });
+  }
```

Comment:
These errors should be handled by a PSQL error handler rather than custom errors. So the query should be made with the data provided by the user and then depending on the error (if returned from the db query) handled accordingly.